File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height; // get height from user
    do
    {
        height = get_int("Enter height -> : ");
    }
    while (height < 1 || height > 8); // end of do while loop
    
    for (int i = 1; i < (height + 1); i++) // for each row
    {
        for (int j = 0; j < (height - i); j++) // print spaces
        {
            printf(" ");
        }
        for (int h = 1; h < (i + 1); h++) // print #
        {
            printf("#");
        }
        printf("\n"); // go to new line
    }
}

File: sentimental-mario-more/mario.py
# get user input
while True:
    try:
        height = int(input("Enter height -> : "))
        if height > 0 and height < 9:
            break
    except:
        print("not valid")

# for each row, r = row
for r in range(height):
    # print spaces, s = spaces
    for s in range(1, height - r):
        print(" ", end="")
    # b = bricks
    for b in range(r + 1):
        print("#", end="")
    # print gaps
    print("  ", end="")
    # print second set of bricks
    for b in range(r + 1):
        print("#", end="")
    # print nothing to go to new line
    print("")
